src/main/java/com/flightstats/hub/channel/TagContentResource.java

```diff
@@ -196,32 +165,30 @@ public Response getValue(@PathParam("tag") String tag,
                              @PathParam("s") int second,
                              @PathParam("ms") int millis,
                              @PathParam("hash") String hash,
-                             @HeaderParam("Accept") String accept
-    ) {
-        long start = System.currentTimeMillis();
-        ContentKey key = new ContentKey(year, month, day, hour, minute, second, millis, hash);
-        ItemRequest itemRequest = ItemRequest.builder()
+                             @HeaderParam("Accept") String accept) {
+        final ContentKey key = new ContentKey(year, month, day, hour, minute, second, millis, hash);
```

Comment:
I think we agreed that we should not be adding final to method scoped variables

Reply:
I cleaned up some more. We will clean up rest as we refactor.